src/components/PermissionsGate/index.js

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+export { PermissionsGate } from './PermissionsGate';
```

Comment:
Should we put this component in the ui repository ? This component is similar to the `PublicRoute` and `PrivateRoute` components.

Reply:
Changes have been done regarding our discussion ;)